src/esbmc/esbmc_parseoptions.h

```diff
@@ -103,6 +103,9 @@ class esbmc_parseoptionst : public parseoptions_baset, public language_uit
 
 public:
   goto_functionst goto_functions;
+
+  // for multi-kind/incr
+  std::set<std::pair<std::string, std::string>> to_remove_claims;
```

Comment:
Same as above: what information is being accumulated here, and does it need to be stored in the `esbmc_parseoptionst` class instance?